imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -17,11 +18,84 @@ import (
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
+// The values have been decided based on running spot tests for different images.
+var imageFamilyBootTimeThresholdMap = map[string]float64{
+	"centos":      60,
+	"debian":      50,
+	"rhel":        60,
+	"rocky-linux": 60,
+	"sles-12":     85,
+	"sles-15":     120,
+	"ubuntu":      75,
+	"ubuntu-pro":  110,
+}
+
 const (
 	markerFile     = "/boot-marker"
 	secureBootFile = "/sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c"
 )
 
+func getThresholdValue(image string) float64 {
+	names := make([]string, 0, len(imageFamilyBootTimeThresholdMap))
+	for name := range imageFamilyBootTimeThresholdMap {
+		names = append(names, name)
+	}
+	sort.Sort(sort.Reverse(sort.StringSlice(names)))
+	for _, name := range names {
+		if strings.Contains(image, name) {
+			return imageFamilyBootTimeThresholdMap[name]
+		}
+	}
+	return 0
+}
+
+func lookForProcesses() error {
```

Comment:
this function is pretty specific to the conditions of the new test - it checks for sshd and guest agent both being up. i suggest either extracting the process listing logic to its own function or at least renaming this function to indicate what it is doing.